File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/V. Comparison/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/V

#include <iostream>
using namespace std;

int main()
{
    int a, b;
    char s;
    cin >> a >> s >> b;
    bool isRight;
    switch (s)
    {
    case '>':
        isRight = a > b; break;
    case '<':
        isRight = a < b; break;
    case '=':
        isRight = a == b; break;
    }

    cout << (isRight? "Right": "Wrong") << endl;
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/V. Comparison/main.dart
import 'dart:io';

void main() {
  List<String> input  = stdin.readLineSync()!.split(" ");

  int a = int.parse(input[0]);
  int b = int.parse(input[2]);

  bool isRight = false;
  switch (input[1])
  {
    case '>':
        isRight = a > b; break;
    case '<':
        isRight = a < b; break;
    case '=':
        isRight = a == b; break;
  }

  print((isRight ? "Right" : "Wrong"));
}
